### ISSUE TITLE
Looking for help with your Discord Theme

### ISSUE THREAD

**Issue by DarkClippy** [PONYMODZ/Discord-stuff#2]
Hello there, BMT! I tried adding you on Discord for PMs, but your code didn't have any results. I just downloaded the theme you made for Better Discord and I was a bit disappointed that it didn't look as the screenshots implied.

My background is still just plain black. Some of the CSS code is working, like the Unicorn emojis on top and the pink accents, but there's no pony image in the background. :(
I've taken the liberty of downloading your master file already, but I don't know how to do CSS so I need some help.

**Comment by the-bmt** [comment]
Yeah it is outdated and it will not get any more updates since discord changed so much and most of it isn't even doable anymore, but I am working on a new version. I can't say when I am done with the new one but it will be available in a few weeks/months
